### PR TITLE
#141404651 Game invite and user limit

### PR DESCRIPTION
**Pull request by thebere**
#### What does this PR do?
Implements the game invite and game user limit requirements.

#### Description of Task to be completed?
1. Limit users to send send invites to at most 11 people other than themselves.
2. Limit the maximum players for each game to 12 people and a minimum of 3.
3. Pop up to inform users of activity.
4. An authenticated endpoint to search current users.

#### How should this be manually tested?
N/A

#### Any background context you want to provide?
To test the send invite funtionality, a gmail address with the pass word should be added as environmental variables in this format:
GMAIL_ADDRR= 
GMAIL_PASSWORD=

#### What are the relevant pivotal tracker stories?
Users should be able to send invites to at most 11 people other than themselves (#141404651)

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -18,6 +18,9 @@
     "new-cap": 0,
     "consistent-return": 0,
     "no-param-reassign": 0,
+    "no-use-before-define": 0,
```
Comment (sdowoade) [inline .eslintrc.json:21]: We should avoid changing this file. The rules are set for a reason.

File: app/controllers/index.js [hunk 2]
```diff
@@ -1,23 +1,20 @@
-/**
- * Module dependencies.
- */
-var mongoose = require('mongoose'),
-    async = require('async'),
-    _ = require('underscore');
+// const mongoose = require('mongoose');
```
Comment (sdowoade) [inline app/controllers/index.js]: We can remove this if it's not being used. Comments should always be explanatory.

File: .eslintrc.json (hunk 1 above)
Comment (thebere) [reply .eslintrc.json:21]: Okay. Noted

File: .env [hunk 3]
```diff
@@ -0,0 +1,5 @@
+SECRETKEY= secrtky41
+MONGOHQ_URL= mongodb://127.0.0.1:27017/kakashi
```
Comment (sdowoade) [inline .env]: This should not be in here.

File: bower_components/angular-bootstrap/.bower.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
+{
+  "author": {
```
Comment (sdowoade) [inline bower_components/angular-bootstrap/.bower.json]: Let's avoid pushing generated modules. This folder should not be here.

**Review by sdowoade — CHANGES_REQUESTED** [review]

**Review by sdowoade — APPROVED** [review]

**Pull request merged**